### PR TITLE
Added hard_wrap: true to MarkDown, and also option for target _blank.

### PR DESCRIPTION
**Pull request by pedrosino**

### CODE REVIEW

File: WcaOnRails/app/helpers/application_helper.rb [hunk 1]
```diff
@@ -27,8 +27,16 @@ def mail_to_wca_board
...
 
-  def md(content)
-    Redcarpet::Markdown.new(Redcarpet::Render::HTML.new).render(content).html_safe
+  def md(content, target_blank: false)
+    options = {
+      hard_wrap: true
+    }
+
+    if target_blank
+      options[:link_attributes] = { rel: 'nofollow', target: "_blank" }
```
Comment (jfly) [inline WcaOnRails/app/helpers/application_helper.rb]: I'm not sure that we want `rel: 'nofollow'` here. According to https://en.wikipedia.org/wiki/Nofollow, it seems to be something we'd use if we let anybody add links to our website, but we only let trusted people like delegates add links.

**Comment by jfly** [comment]
What's the point of the `target_blank` argument if nobody is using it?

**Comment by pedrosino** [comment]
I will use in the competition index, for the venue links.

File: WcaOnRails/app/helpers/application_helper.rb (hunk 1 above)
Comment (pedrosino) [inline WcaOnRails/app/helpers/application_helper.rb]: Ok, will remove.

**Comment by jfly** [comment]
Ok, that change should go in that PR, then, but this is fine.

**Pull request merged**